iroh-gateway/src/rpc.rs

```diff
@@ -1,16 +1,29 @@
 use std::result;
 
 use anyhow::Result;
+use futures::stream::{BoxStream, StreamExt};
 use iroh_rpc_client::{create_server, GatewayServer, ServerError, ServerSocket};
 use iroh_rpc_types::gateway::{
-    GatewayAddr, GatewayRequest, GatewayService, VersionRequest, VersionResponse,
+    GatewayAddr, GatewayRequest, GatewayService, VersionRequest, VersionResponse, WatchRequest,
+    WatchResponse,
 };
 use tracing::info;
 
 #[derive(Default, Debug, Clone)]
 pub struct Gateway {}
 
 impl Gateway {
+    #[tracing::instrument(skip(self))]
+    fn watch(self, _: WatchRequest) -> BoxStream<'static, WatchResponse> {
+        async_stream::stream! {
+            loop {
+                yield WatchResponse { version: env!("CARGO_PKG_VERSION").to_string() };
+                tokio::time::sleep(std::time::Duration::from_millis(1000)).await;
```

Comment:
`Duration::from_secs(1)` reads a little easier :) also would be nice to put into a constant